doc/source/howtos/virt_sandbox.rst

```diff
@@ -11,20 +11,20 @@ However, it should apply to any virtualization technology
 (for example, VirtualBox) with any Linux distribution.
 
 This manual requires three virtual machines for |PXC| nodes,
-and one for HAProxy client, which redirects requests to the nodes.
-Running HAProxy on an application server,
+and one for the ProxySQL, which redirects requests to the nodes.
+Running ProxySQL on an application server,
 instead of having it as a dedicated entity,
 removes the unnecessary extra network roundtrip,
 because the load balancing layer in |PXC| scales well
 with application servers.
 
 1. Install |PXC| on the three cluster nodes, as described in :ref:`yum`.
 
-#. Install HAProxy and ``sysbench`` on the client node:
+#. Install :ref:`ProxySQL <load_balancing_with_proxysql>` and ``sysbench`` on the client node:
 
    .. code-block:: bash
 
-      yum -y install haproxy sysbench
+      yum -y install proxysql2 sysbench
 
 #. Make sure that the :file:`my.cnf` configuration file on the first node
    contains the following::
```

Comment:
We should add a note about `pxc-encrypt-cluster-traffic` change.

Reply:
Is this change applied to 5.7 too? This pull request is for 5.7. The tickets related to cluster traffic encryption were mostly limited to 8.0

Reply:
This is not applicable to 5.7.

Reply:
@rameshvs02 This pull request is for v5.7. Has the default value changed for this version too?

Reply:
@borys-belinsky  No, ignore `pxc-encrypt-cluster-traffic` note for v5.7.